[3.4.0RC1] NoClassDefFoundError via FilesSyncHelper.scheduleFilesSyncIfNeeded (Android 9)

via GPlay, Android 9 only (at least until now), 3.4.0RC1
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:3520)
  at android.app.ActivityThread.access$1300 (ActivityThread.java:200)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1667)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:193)
  at android.app.ActivityThread.main (ActivityThread.java:6680)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:858)
Caused by: java.lang.ClassNotFoundException: 
  at java.lang.VMClassLoader.findLoadedClass (Native Method)
  at java.lang.ClassLoader.findLoadedClass (ClassLoader.java:738)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:363)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:312)
  at android.app.AppComponentFactory.instantiateService (AppComponentFactory.java:103)
  at android.support.v4.app.CoreComponentFactory.instantiateService (CoreComponentFactory.java:68)
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:3515)
Caused by: java.lang.NoClassDefFoundError: 
  at com.evernote.android.job.GcmAvailableHelper.isGcmApiSupported (Unknown Source)
  at com.evernote.android.job.JobApi.isSupported (JobApi.java:125)
  at com.evernote.android.job.JobProxy$Common.cleanUpOrphanedJob (JobProxy.java:293)
  at com.evernote.android.job.JobManager.schedule (JobManager.java:186)
  at com.evernote.android.job.JobRequest.schedule (JobRequest.java:436)
  at com.owncloud.android.utils.FilesSyncHelper.scheduleFilesSyncIfNeeded (FilesSyncHelper.java:252)
  at com.owncloud.android.MainApp.initSyncOperations (MainApp.java:312)
  at com.owncloud.android.MainApp.onCreate (MainApp.java:167)
  at android.app.Instrumentation.callApplicationOnCreate (Instrumentation.java:1154)
  at android.app.ActivityThread.handleBindApplication (ActivityThread.java:5882)
  at android.app.ActivityThread.access$1100 (ActivityThread.java:200)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1651)
Caused by: java.lang.ClassNotFoundException: 
  at dalvik.system.BaseDexClassLoader.findClass (BaseDexClassLoader.java:134)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:379)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:312)
```

--- a/build.gradle
+++ b/build.gradle
@@ -221,7 +221,7 @@ dependencies {
     implementation 'com.github.albfernandez:juniversalchardet:2.0.0' // need this version for Android <7
     implementation 'com.google.code.findbugs:annotations:2.0.1'
     implementation 'commons-io:commons-io:2.6'
-    implementation 'com.github.evernote:android-job:v1.2.5'
+    implementation 'com.github.tobiaskaminsky:android-job:v1.2.6.1' // 'com.github.evernote:android-job:v1.2.5'
     implementation 'com.jakewharton:butterknife:9.0.0-rc2'
     annotationProcessor 'com.jakewharton:butterknife-compiler:9.0.0-rc2'
     implementation 'org.greenrobot:eventbus:3.1.1'

